subsys/bluetooth/controller/hci_driver.c

```diff
@@ -979,9 +979,12 @@ static int configure_supported_features(void)
 	}
 
 	if (IS_ENABLED(CONFIG_BT_CTLR_CHANNEL_SOUNDING)) {
-		err = sdc_support_channel_sounding_test();
-		if (err) {
-			return -ENOTSUP;
+		if (IS_ENABLED(CONFIG_BT_CTLR_CHANNEL_SOUNDING_TEST))
```

Comment:
think it's a bit cleaner to move it out of this if block `if (IS_ENABLED(CONFIG_BT_CTLR_CHANNEL_SOUNDING)) {`

the dependency is enforced by kconfig